www/js/app.js

```diff
@@ -218,19 +220,88 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
         return true;
     });
     // Top menu :
+    function removeClasses(){
+        $('#about').removeClass("fadeIn_L").removeClass("fadeOut_R");
+        $('#configuration').removeClass("fadeIn_L").removeClass("fadeIn_R").removeClass("fadeOut_L").removeClass("fadeOut_R");
+        $('#articleContent').removeClass("fadeIn_R").removeClass("fadeOut_L");
+    }
```

Comment:
I think this function should be renamed to `removeAnimationClasses` to be clear about what classes are being removed. I think you can easily move this function and the next one to `uiUtil.js` to help declutter `app.js`. Please also add spaces before open braces (curly brackets).

Reply:
Ok sure.... yes that makes more sense also